src/sshkeys/sshkeys.controller.ts

```diff
@@ -0,0 +1,66 @@
+import { Body, Controller, Delete, Get, HttpException, HttpStatus, Param, Post } from '@nestjs/common';
+import { SshKeysService } from './sshkeys.service';
+import { AdminOnly } from 'src/auth/decorators/adminonly.decorator';
+import { CreateSshKeyDto } from './dto/create-sshkey.dto';
+import { GetUser } from 'src/auth/decorators/user.decorator';
+import { UserDecoratorType } from 'src/auth/types/user-decorator.type';
+
+@Controller('sshkeys')
+export class SshKeysController {
+    constructor(
+        private sshkeysService: SshKeysService
+    ) {}
+
+    // POST /sshkeys/my
+    // This action adds a key ${setSshDto} to a user's keys
+    @Post('my')
+    async createSshKey(@Body() createSshDto: CreateSshKeyDto, @GetUser() user: UserDecoratorType) {
+        try {
+            if (!await this.sshkeysService.createSshKey(user.sub, createSshDto)) {
+                return new HttpException("Unable to add this ssh key. Please verify the key and name.", HttpStatus.BAD_REQUEST);
+            }
+            return "SSH key was successfully created."
```

Comment:
You send back a plain text and not json?

same in `deleteSshKey()`

Reply:
@BioTheWolff  has fixed that, it returns a json now : 
```json
{
  "status": "OK / removed"
  } 
```

Reply:
For the functions that return a list of ssh keys, they have been altered to follow the same schema for simplicity : 
```json
{
            "status": "OK",
            "content":  [sshkeys]
}
```

Reply:
I don't understand why you have added a `statusCode`, we already have this information in the response status code

Reply:
Now it's `status` instead of `statusCode` with string and not number like the error nesjs returns?

Also you don't need the `"` in js to have objects/json serialization.

Reply:
Yes, `status` returns a brief information of what was done. `statusCode` can be inferred from the HTTP status code.
The quotes have been removed, thanks.